ush/mesoscale/mesoscale_production_restart.py

```diff
@@ -20,20 +23,33 @@
 NET = os.environ['NET']
 RUN = os.environ['RUN']
 COMPONENT = os.environ['COMPONENT']
-VERIF_CASE = os.environ['VERIF_CASE']
 STEP = os.environ['STEP']
+VERIF_CASE = os.environ['VERIF_CASE']
 
 # Copy files for restart
-if STEP == 'plots':
+if STEP == 'stats':
+    VERIF_CASE = os.environ['VERIF_CASE']
+    RESTART_DIR = os.environ['RESTART_DIR']
+    working_dir = os.path.join(DATA, VERIF_CASE)
+    completed_jobs_file = os.path.join(RESTART_DIR, 'completed_jobs.txt')
+    if os.path.exists(RESTART_DIR):
+        if (os.path.exists(completed_jobs_file) 
+                and os.stat(completed_jobs_file).st_size != 0):
+            print(f"Copying restart directory {RESTART_DIR} "
+                  +f"into working directory {working_dir}")
+            cutil.run_shell_command(
+                ['cp', '-rpv', RESTART_DIR, working_dir]
+            )
+elif STEP == 'plots':
     COMOUTplots = os.environ['COMOUTplots']
     RESTART_DIR = os.environ['RESTART_DIR']
-    COMPLETED_JOBS_FILE = os.environ['COMPLETED_JOBS_FILE']
-    working_dir = os.path.join(DATA, VERIF_CASE, 'out')
-    completed_jobs_file = os.path.join(RESTART_DIR, COMPLETED_JOBS_FILE)
+    SAVE_DIR = os.environ['SAVE_DIR']
+    completed_jobs_file = os.path.join(RESTART_DIR, f'completed_jobs.txt')
     if os.path.exists(completed_jobs_file):
         if os.stat(completed_jobs_file).st_size != 0:
             cutil.run_shell_command(
-                ['cp', '-rpv', os.path.join(RESTART_DIR,'*'), working_dir]
+                ['cp', '-rpv', os.path.join(RESTART_DIR,'*'), SAVE_DIR]
             )
 
+
 print("END: "+os.path.basename(__file__))
```

Comment:
@malloryprow Before you review, I just noticed the differences in this particular file seem to indicate that I copied over the pre-mpmd version of this script that doesn't have the working directory stuff for mpmd.  I'm not terribly sure if it makes any difference, but I wanted to do a quickie test using the updated restart file to know that we have the correct restart script with mpmd.  Hold a quick sec while I make this test.

Reply:
It didn't matter which version of this py script was used, the final result was the same size of final stat file compared to the stat file without restart.  I think that this is the more correct version since I think changes were made in this file in a previous recent PR.

Reply:
@malloryprow I need to make further updates to this file.  Turns out that the old one was actually correct.

Reply:
Updated to the new file, which is the correct one.  We can resolve this particular conversation.